lib/fennec/udp/worker_supervisor.ex

```diff
@@ -4,14 +4,14 @@ defmodule Fennec.UDP.WorkerSupervisor do
 
   alias Fennec.UDP
 
-  def start_link(base_name) do
+  def start_link(opts, base_name) do
```

Comment:
There is one little issue. This will make `opts` to be second argument in `worker`, since this will be the last _default_ argument. But all variable arguments for the `worker` will obviously follow the `opts`